cmake/prerelease.lock

```diff
@@ -0,0 +1 @@
+$Format:%d$
```

Comment:
This is no longer supposed to be a template.

> We create a ``prerelease.txt`` file in the repository, but not in the repository root. I suggest ``cmake/prerelease.lock``. The contents can be something like ``If this file is present, a prerelease will be built by default. You can override this with cmake options ...<mention cmake options below>...``

Here's my suggestion for the content:
```suggestion
When this file is present, an executable that identifies as a pre-release is built by default.
If you want a release build instead, add `SOL_FORCE_RELEASE=On` to your `cmake` command to override the default.
If you want to customize the pre-release part in the version string, you can do it via the `SOL_PRERELEASE_STRING` CMake option.
```
**EDIT**: Replaced `SOLC_` prefix with `SOL_` (see comments below).